index.html

```diff
@@ -1,12 +1,50 @@
+<!-- 
+commit 1 - forked repo
+commit 2 - setup
+commit 3 - blank project
+commit 4 - create files
+commit 5 - final commit 
+-->
+
 <!DOCTYPE html>
 <html lang="en">
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Weather Report</title>
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
+    <link rel='stylesheet' href='/styles/index.css'>
 </head>
 <body>
-    
+    <div class='overlay'> </div>
+    <main class='container'>
+        <h1 id='h1'>Yay Area</h1>
+        <section>
+            <h2>Temperature (&deg;F)</h2>
+            <button id='up' class="btn btn-outline-dark btn-sm">Up</button>
+            <button id='down' class="btn btn-outline-dark btn-sm">Down</button>
+            <span id='temperature'>40</span>
+        </section>
+        <section>
+            <h2>Sky</h2>
+            <label for='weather'></label>
+            <select name='weather' id='sky-menu'>
+                <option value='null'></option>
+                <option value='Sunny' id='sunny'>Sunny</option>
+                <option value='Cloudy' id='cloudy'>Cloudy</option>
+                <option value='Rainy' id='rainy'>Rainy</option>
+            </select>
+        </section>
+        <section>
+            <h2>Location</h2>
+            <input type='text' placeholder='Rep your city' id='city-text'></input>
+            <button type='reset' id='city-button' class="btn btn-outline-dark btn-sm">Reset</button>
+        </section>
+        <div class='landscape'>
+            <img class='landscape' src='/pngwing.com.png'>
+        </div>
```

Comment:
I would put a section around your landscape so you can add styles to it and manipulate the contents